runtime/jcl/common/stdinit.c

```diff
@@ -155,21 +145,29 @@ standardInit( J9JavaVM *vm, char *dllName)
 	 */
 #ifdef J9VM_OPT_SIDECAR
 #if !defined(J9VM_INTERP_MINIMAL_JCL)
-	if (result == 0) {
+	{
 		UDATA handle = 0;
 		result = (jint)vmFuncs->registerBootstrapLibrary(vm->mainThread, "zip", (J9NativeLibrary **)&handle, FALSE);
 	}
 #endif /* !J9VM_INTERP_MINIMAL_JCL */
 #endif /* J9VM_OPT_SIDECAR */
 
 	vmFuncs->internalAcquireVMAccess(vmThread);
+	
+	if (0 == result) {
```

Comment:
Could move this result check before the internalAcquireVMAccess() and put everything to the internalReleaseVMAccess() inside the block, since nothing is going to run if result != 0.

Reply:
Already updated the code as suggested above.